lib/Compiler.js

```diff
@@ -321,7 +321,7 @@ Compiler.prototype.emitAssets = function(compilation, callback) {
 				}
 				var content = source.source();
 				if(!Buffer.isBuffer(content))
```

Comment:
I would force braces with ifs (separate PRs) as the form without can hide errors easily.